package.json

```diff
@@ -70,6 +70,8 @@
     "karma-coverage-istanbul-reporter": "^1.3.0",
     "karma-jasmine": "^1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
+    "ncp": "^2.0.0",
+    "node-sass": "^4.9.2",
```

Comment:
unnecessary dependencies. `node-sass` is installed with the Angular cli. The copy can be handled by the rollup build file.